src/frontend/js/api/mocks/joanie/organizations.ts

```diff
@@ -0,0 +1,44 @@
+import { rest } from 'msw';
+import { getAPIEndpoint } from 'api/joanie';
+
+export interface OrganizationMock {
+  id: string;
+  code: string;
+  title: string;
+  cover: {
+    sizes: string;
+    src: string;
+    srcset: string;
+  };
```

Comment:
Will look something like this :
```suggestion
  logo: {
    filename: string;
    url: string;
    height: number;
    width: number;    
  };
```

Reply:
Ok but something sure is that the property name will be `logo`.

Then currently, we already return the organization logo in admin endpoint with the type I wrote.